### DISCUSSION TITLE
Quarto Kickstarter Issue

### DISCUSSION THREAD

**Discussion by ivanuricardo** [quarto-dev/quarto-nvim#95] [Q&A] [answered]
Hello! I am trying to install the Kickstarter at the moment but get an error that the config is not running. More specifically, I git clone the config and when I enter "nvim" in the terminal, mostly everything installs except at the end I get the error:

Failed to run `config` for indent-blankline.nvim

.../share/nvim/lazy/indent-blankline.nvim/lua/ibl/utils.lua:18: 'show_current_context_start' is not a valid key
 of ibl.config

I can navigate the start menu, but this seems like an issue with the config? 

Thanks in advance for any help!

**Comment by jmbuhr** [comment, marked as answer]
Fixed it :)
https://github.com/jmbuhr/quarto-nvim-kickstarter/commit/8413978ad750b40c599e8f1b6b15a15f964ef01b
